Makefile

```diff
@@ -28,6 +28,32 @@ all: $(RUNC_LINK)
 static: $(RUNC_LINK)
 	CGO_ENABLED=1 go build -i -tags "$(BUILDTAGS) cgo static_build" -ldflags "-w -extldflags -static -X main.gitCommit=${COMMIT} -X main.version=${VERSION}" -o runc .
 
+release: $(RUNC_LINK)
+	@flag_list=(seccomp selinux apparmor static); \
```

Comment:
Put them into a separate script will make code clean, but the script need to access some variable in Makefile($COMMIT, $VERSION), we have some way to do it, but each way will make code complex:
1: Pass these variable as script's argument, or export them for script
    we need update the argument when we add/modify variable.
2: Let the script parse these variable in Makefile
    complex
3: clone these variable into script
    hard to maintenance
So, maybe put the ugly script into Makefile seems better than other selection...